backend/src/api/users/users.controllers.js

```diff
@@ -0,0 +1,24 @@
+const UserDB = require('./user.model');
+const catchAsync = require('../../utils.js/catchAsync');
+
+const updateUsersController = catchAsync(async (req, res, next) => {
+  if (req.user && req.body) {
```

Comment:
you would always have req.user if authorize middleware passed execution to this middleware